web/components/SchedulePage.jsx

```diff
@@ -1,33 +1,143 @@
 import React from 'react';
+import moment from 'moment';
+import ScheduleHeader from './ScheduleHeader';
+import SideBar from './SideBar';
+import Flex from './Flex';
+import SetBlock from './SetBlock';
+import Text from './Text';
 
 export default class SchedulePage extends React.Component {
-    renderBaseOnParams = (match) => {
-        if (match.params.teamMemberId) {
-            return (
-                <h6>
-                teamMemberId:
-                    {match.params.teamMemberId}
-                </h6>
-            )
+
+    constructor(props) {
```

Comment:
With ES6 and newer React versions, it's no longer necessary to use the `constructor` function (the setlife.js templates are outdated...). We are going to move these state structures into Redux, but for informational purposes. The new way to define local, non-Redux state in a component would look like this:

```
export default class SchedulePage extends React.Component {
    state = {
        daysOfWeek: [],
            selectedDay: moment.now(),
            setBlocks: [{
                id: '1',
                date: '12/12/18',
                blockTime: 'Setblock 1 (12am - 4am)',
                blockFraction: '1.0',
                issueUrl: 'http://issueURL',
                description: 'A mocked setBlock',
                teamMember: '',
            }, {
                id: '2',
                date: '13/12/18',
                blockTime: 'Setblock 2 (4:30am - 8:30am)',
                blockFraction: '0.5',
                issueUrl: 'http://issue2URL',
                description: 'A mocked setBlock2 Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse tellus metus, scelerisque id nisl nec, ',
                teamMember: '',
            }
            ]
    }
    ....
    render {
    ...
}
```